perfdash/test/jobs.yaml

```diff
@@ -0,0 +1,7 @@
+periodics:
+- cron: '1 17 * * *' # Run daily at 9:01PST (17:01 UTC)
+  name: ci-kubernetes-e2e-gce-scale-performance
+  tags:
+  - "perfDashPrefix: gce-5000Nodes"
+  - "perfDashBuildsCount: 270"
```

Comment:
I think we could accelerate running `make run` without the `perfDashBuildsCount` override, WDYT?

Reply:
Done.